### ISSUE TITLE
Keyboard Shortcuts for switching Fullscreen modes quickly

### ISSUE THREAD

**Issue by Arxareon** [dralletje/Windowed#36]
Hey!

I'm using Chrome, I'd really love to have a small feature to be able to change fullscreen modes quickly via browser shortcuts. Currently, only the default open the extension popup keyboard shortcut is present which is not a big help because I still have to navigate up and change the setting manually.

Otherwise, very nice tool!

**Comment by HatWearingMann** [comment]
Yeah, it'd be great if the menu only appeared upon pressing the fullscreen button or the [F] key with [SHIFT] held down.

**Comment by Arxareon** [comment]
> Yeah, it'd be great if the menu only appeared upon pressing the fullscreen button or the [F] key with [SHIFT] held down.

That's not what I mean. I was suggesting keybinding to apply certain fullscreen modes instantly. Having the ask window pop up based on pressing Shift for instance would not be the best solution.

**Comment by HatWearingMann** [comment]
Why not both? :-)

**Comment by omerfarukatas** [comment]
Hi, I think my request is similar to Arxareon's. It would be great if we could use to shortcuts without first having to open the pop-up menu. For example, when on the page of a YouTube video, I could press "i" and open the in-window view immediately, without having to deal with the pop-up menu at all.

In any case, I love the extension! People like you make life easier. I am currently learning JavaScript. I would love to create browser extensions myself some day.

**Comment by dralletje** [comment]
> Hi, I think my request is similar to Arxareon's. It would be great if we could use to shortcuts without first having to open the pop-up menu. For example, when on the page of a YouTube video, I could press "i" and open the in-window view immediately, without having to deal with the pop-up menu at all.

I can't do this, sadly. This extension is website independent, and I can't beforehand know how a website wants to be "fullscreened". So Best I can do is pressing whatever the fullscreen shortcut for a website is, followed by pressing `i` when the popup is open. (Every option has a shortcut when the popup is open)

**Comment by rhys-saldanha** [comment]
> Every option has a shortcut when the popup is open

Please can you underline the letter in each option that maps to the shortcut key? It took me an embarrassingly long amount of time to figure out that there are shortcuts!

ie: 
![image](https://github.com/user-attachments/assets/e451a2c5-be86-4527-ae6c-ba22bb4a2bc1)

**Comment by death0serpent** [comment]
This seems related to my recent question in [#59](https://github.com/dralletje/Windowed/pull/59). Apologies but I'm quite utterly confused. Are there any other extensions that might interfere with the functionality that seems to have been implemented according to the comments in #59 ?

**Comment by dralletje** [comment]
> Please can you underline the letter in each option that maps to the shortcut key? It took me an embarrassingly long amount of time to figure out that there are shortcuts!
 
@rhys-saldanha I don't want to clutter the UI too much like that for users who clicked, as they are in my eyes not likely to switch to a shortcut key then. But I did make it very clear for people who opened the dialog with a shortcut key:

<img width="379" alt="Image" src="https://github.com/user-attachments/assets/8745f55a-581c-4616-8732-e0ae98386a44" />

**Comment by rhys-saldanha** [comment]
Yup that makes perfect sense! 

I'll double check when I'm back at a computer, but I'm pretty sure that I don't see the keyboard shortcuts when opening the popup in YouTube using the 'f' key. Me not seeing the shortcuts in the pop-up would explain why it took me a while to even find out that keyboard shortcuts exist.

**Comment by dralletje** [comment]
> but I'm pretty sure that I don't see the keyboard shortcuts when opening the popup in YouTube using the 'f' key

I only just released it this week! If you have Windowed 33 you should have it...
Also I only released that version for Chrome... I'll try to make a release for Firefox next week

**Comment by rhys-saldanha** [comment]
Yup that makes perfect sense! 

I'll double check when I'm back at a computer, but I'm pretty sure that I don't see the keyboard shortcuts when opening the popup in YouTube using the 'f' key. Me not seeing the shortcuts in the pop-up would explain why it took me a while to even find out that keyboard shortcuts exist.

**Comment by HyperionCoding** [comment]
@dralletje When is the update coming to Firefox?
